requirements.txt

```diff
@@ -1,2 +1,3 @@
 numpy>=1.8.2
 scipy>=0.13.3
+pytest>=3.7.4
```

Comment:
I think that we need a separate `requirements.txt` file for development/ developers. You only need `pytest` if you want to develop the package and not if you want to use it.

Reply:
Please check how it is done in other packages.

Reply:
I don't think it's needed as long as in the continuous integration tools we specify to install pytest, codecov and pytest-cov. 
One way around this is to have a requirements directory which has requirements.txt and requirements-dev.txt in. Then in the base directory you would have requirements.txt which specifies to use the normal requirements, then with devs you would just do
'''
pip install -r requirements/requirements-dev.txt
'''
This way the normal command still works for non devs.

Reply:
Great. This will be resolved by #5.